resources/public/api/conf/P1.0/schemas/child-tax-credits-response.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Individual matching request",
+  "type": "object",
+  "properties": {
+    "applications": {
+      "description": "The applications that are active during the date range requested",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "description": "The identifier for the household application that the individual is active on. Multiple individuals can be active on the same application id.",
+            "example": 123
+          },
+          "awards": {
+            "type": "array",
+            "description": "The awards that were active for this application during the time period",
+            "items": {
+              "type": "object",
+              "properties": {
+                "payProfCalcDate": {
+                  "type": "string",
+                  "description": "Payment Profile Calculated Date",
+                  "example": "1996-08-01"
+                },
+                "totalEntitlement": {
+                  "type": "number",
+                  "description": "The sum of all the award entitlements for an award version; the net amount the entitlement unit will receive",
```

Comment:
Do we need to add a range to the description for monetary amounts as I have done in Income API docs?